pkg/platformconfig/types.go

```diff
@@ -278,6 +279,15 @@ type IngressConfig struct {
 	Oauth2ProxyURL             string   `json:"oauth2ProxyURL,omitempty"`
 }
 
+type ElasticSearchConfig struct {
```

Comment:
This needs to be added to `configuring-a-platform.md` with explanations, but it can be done after everything is implemented.

Reply:
@TomerShor yes, docs will be covered in a separate pr